### PR TITLE
ICDS Dashboard: Adding "not launched" to map legend

### PR DESCRIPTION
**Pull request by saiprakash-v**
https://app.asana.com/0/1172925702295169/1173537631663916
##### SUMMARY

This pr adds "Not Launched" to the fills of the map Legend

### CODE REVIEW

**Comment by saiprakash-v** [comment]
@Rohit25negi This is ready for review

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
hey looks good. Should this change be under FF?

**Comment by saiprakash-v** [comment]
> hey looks good. Should this change be under FF?

Yes. Just got confirmation from Nitish. I will update

**Comment by Rohit25negi** [comment]
@saiprakash-v is it ready to next round of review
?

**Comment by saiprakash-v** [comment]
> @saiprakash-v is it ready to next round of review
> ?

Yes, I was waiting to inform you after the build succeed

File: custom/icds_reports/reports/adolescent_girls.py [hunk 1]
```diff
@@ -64,6 +64,7 @@ def get_data_for(filters):
 
     fills = OrderedDict()
     fills.update({'Adolescent Girls': MapColors.BLUE})
+    fills.update({'Not Launched': MapColors.GREY}) if beta else None
```
Comment (Rohit25negi) [inline custom/icds_reports/reports/adolescent_girls.py:67]: could this simply be.
```
if beta:
    fills.update({'Not Launched': MapColors.GREY})

```

**Review by Rohit25negi — COMMENTED** [review]
its almost there.  left one comment which is not blocking but good to have. I think single line if statements are usually used when we want to assign the return to some variable. In case of this, no need of that and we can simply use nromal if statement.

**Comment by Rohit25negi** [comment]
Also, Did you test this locally. Both, the FF is working and the change you expect is working.

**Comment by saiprakash-v** [comment]
> Also, Did you test this locally. Both, the FF is working and the change you expect is working.

It was working until I merged master and resolved merge conflicts. Must have messed up something :facepalm: . Looking into it

**Comment by Rohit25negi** [comment]
is this good for next round of review? @saiprakash-v

**Comment by saiprakash-v** [comment]
> is this good for next round of review? @saiprakash-v

Yes @Rohit25negi

**Review by Rohit25negi — APPROVED** [review]
Looks good. Did you test it locally with/without feature flag for all the pages you made the change on?

**Comment by saiprakash-v** [comment]
> Looks good. Did you test it locally with/without feature flag for all the pages you made the change on?

Just tested it after updating with latest master code. It is looking good on all the pages with/without feature flag

**Pull request merged**